Asterix/wfsc/wf_control_functions.py

```diff
@@ -62,10 +190,10 @@ def create_interaction_matrix(testbed: Testbed,
         in both case, if the DMs are not initially flat (non zero initial_DM_voltage), 
         we do not make the small phase assumption for initial DM phase
 
-    input_wavefront: 1D-array real
+    initial_DM_voltage: 1D-array real
         initial DM voltage for all DMs
     
-    input_wavefront: 2D-array complex
+    input_wavefront: 
```

Comment:
Was the deletion of this type in the docstring on purpose?

Reply:
yeah I probably wanted to changed it and forgot